build/build.cake

```diff
@@ -118,7 +118,7 @@ Task("Version")
     var installSettings = new NuGetInstallSettings {
         ExcludeVersion  = true,
         Prerelease = true,
```

Comment:
This should probably be false now, right?

Reply:
Removed